packages/showcase/webpack.config.js

```diff
@@ -23,10 +25,7 @@ const config = isProd
       },
 
       resolve: {
-        alias: {
-          // 'index':path.join(__dirname,'..', 'src', 'index.js'),
-          // 'theme':path.join(__dirname,'..', 'src', 'theme.js')
-        }
+        modules: ['node_modules', path.join(__dirname,'..', 'react-vis', 'src')],
```

Comment:
urgh, we should avoid custom node_modules resolutions, it's bound to cause headaches late.
I thought yarn workspaces would allow us to avoid this.

Reply:
Completely agree.
`yarn workspaces`  will definately help us with this, however there is a bunch of cleanup that has to be done first.

Next step (separate PR) is to clean up the imports in the `react-vis`, then we should be able to remove this module resolution.